concent_api/concent_golem_messages/tests/factories.py

```diff
@@ -0,0 +1,69 @@
+import random
+
+from ethereum.utils import denoms
+from golem_messages.factories.helpers import MessageFactory
+import factory.fuzzy
+
+from concent_golem_messages.message import NonceAbstractMessage
+from concent_golem_messages.message import SignedTransaction
+from concent_golem_messages.message import TransactionRejected
+from concent_golem_messages.message import TransactionSigningRequest
+
+
+class NonceAbstractMessageFactory(MessageFactory):
+    class Meta:
+        model = NonceAbstractMessage
+
+    nonce = factory.LazyFunction(
+        lambda: random.randint(0, denoms.turing)
+    )
+
+
+class TransactionAbstractMessage(NonceAbstractMessageFactory):
+    class Meta:
+        model = NonceAbstractMessage
+
+    gasprice = factory.LazyFunction(
+        lambda: random.randint(0, denoms.turing)
+    )
+    startgas = factory.LazyFunction(
+        lambda: random.randint(0, denoms.turing)
+    )
+    to = factory.fuzzy.FuzzyText(length=20, chars='0123456789abcdef')
+    value = factory.LazyFunction(
+        lambda: random.randint(0, denoms.turing)
+    )
+    data = b''
+
+
+class TransactionSigningRequestFactory(TransactionAbstractMessage):
+    class Meta:
+        model = TransactionSigningRequest
+
+    # pylint: disable=no-self-argument
+
+    @factory.post_generation
+    def arct_report_computed_task(msg, _create, _extracted, **kwargs):  # pylint: disable=unused-argument
```

Comment:
@kbeker https://github.com/golemfactory/concent/pull/687#discussion_r207899210
> Please write full name, or comment what this function does :)

The name was wrong, so thanks for catching this, I have changed it to more proper one and added a comment why it has to be done through method instead of setting this directly on class attributes. You can check what this decorator does directly here http://factoryboy.readthedocs.io/en/latest/reference.html#factory.post_generation. In short, it allows you to run some code on the newly created object.
:+1: done